src/ioctls/vm.rs

```diff
@@ -2513,6 +2554,20 @@ mod tests {
         assert!(vm.check_extension(Cap::MpState));
     }
 
+    #[test]
+    fn test_check_extension_int() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        assert!(vm.check_extension_int(Cap::MpState) > 0);
+    }
+
+    #[test]
+    fn test_check_extension_raw() {
+        let kvm = Kvm::new().unwrap();
+        let vm = kvm.create_vm().unwrap();
+        assert!(vm.check_extension_raw(Cap::MpState as c_ulong) > 0);
```

Comment:
IMHO this is literally a `check_extension` test which gets expanded, if we intend to test `check_extension_raw` positively, we can do something like `assert_eq!(vm.check_extension_raw(Cap::MaxVcpus), 1024);` here to state that we want an `int` which its value matters.

Or else we just do negative test here :)

Reply:
1024 here is for RISC-V, as of the other two archs, we will need to set its value accordingly